File: simples.py
import openpyxl

def extrair_dados_planilha(nome_arquivo, nome_planilha):
    # Carregar o arquivo Excel
    workbook = openpyxl.load_workbook(nome_arquivo)

    # Selecionar a planilha pelo nome
    planilha = workbook[nome_planilha]

    # Iterar sobre as linhas da planilha
    for linha in planilha.iter_rows(min_row=2, values_only=True):
        # Supondo que a primeira coluna contém dados importantes
        dado = linha[0]
        print(f"Dado extraído: {dado}")

    # Fechar o arquivo Excel
    workbook.close()

# Substitua 'seu_arquivo.xlsx' pelo nome do seu arquivo Excel
# Substitua 'SuaPlanilha' pelo nome da sua planilha
extrair_dados_planilha('planilhaQuestoes.xlsx', 'Planilha1')

File: simples.js
// Certifique-se de incluir a biblioteca xlsx
// Você pode instalá-la via npm: npm install xlsx
const XLSX = require('xlsx');

function extrairDadosPlanilha(nomeArquivo, nomePlanilha) {
    // Ler o arquivo Excel
    const workbook = XLSX.readFile(nomeArquivo);

    // Selecionar a planilha pelo nome
    const planilha = workbook.Sheets[nomePlanilha];

    // Converter a planilha para um objeto JSON
    const dados = XLSX.utils.sheet_to_json(planilha, { header: 1 });

    // Iterar sobre as linhas do objeto JSON
    for (const linha of dados.slice(1)) {
        // Supondo que a primeira linha contém os cabeçalhos
        const pergunta = linha[0];
        const alternativas = linha.slice(1, -1); // Excluir a última coluna (resposta)

        // Aqui você pode realizar as operações desejadas com os dados
        console.log(`Pergunta: ${pergunta}`);
        console.log(`Alternativas: ${alternativas}`);
    }
}

// Substitua 'planilhaQuestoes.xlsx' pelo nome do seu arquivo Excel
// Substitua 'Planilha1' pelo nome da sua planilha
extrairDadosPlanilha('planilhaQuestoes.xlsx', 'Planilha1');
